_includes/catalogpage.html

```diff
@@ -15,7 +15,7 @@
         {% endfor %}
     </ul>
 
-    <p>See more widgets in the <a href="..">Flutter widget catalog</a>.</p>
+    <p>See more widgets in the <a href="/widgets/widgetindex/">Flutter widget catalog</a>.</p>
```

Comment:
This line appears twice (once at the bottom and once at the top, with the same text), and I think it was intended to take the reader back to the overview of the categories.